workflow/rules/assembly.smk

```diff
@@ -14,12 +14,59 @@ rule assembly:
     conda:
         "../envs/megahit.yaml"
     shell:
-        "(megahit -1 {input.fastq1} -2 {input.fastq2} --out-dir {params.outdir} -f && mv {params.outdir}/final.contigs.fa {output.contigs} ) 2> {log}"
+        "(zcat {input.fastq1} |wc -l > {output.read_count} && "
```

Comment:
The counting snippet duplicates with the metaspades rule below. I'd rather make this a separate rule.

Reply:
generated separate rule